snuba/web/views.py

```diff
@@ -187,7 +187,11 @@ def handle_invalid_dataset(exception: InvalidDatasetError) -> Response:
 def handle_invalid_query(exception: InvalidQueryException) -> Response:
     # TODO: Remove this logging as soon as the query validation code is
     # mature enough that we can trust it.
-    logger.warning("Invalid query", exc_info=True)
+    metrics.increment("query.invalid")
```

Comment:
I would not add this metric. 
We are already tracking invalid queries `snuba.api.query.count` (with a status tag) 

https://github.com/getsentry/snuba/blob/master/snuba/querylog/__init__.py#L12-L65
That separates success/internal failures/user failures.

Logging this here is going to miss the handle_invalid_json case (a different handler) and it would not count subscriptions queries (that do not go through this handler), so it would end up being confusing.